shared/docs/PROTOCOL_STATE_HASH.md

```diff
@@ -106,14 +106,14 @@ graph TD
     D[Did I get any prepareQCs?] --> |Find highest valid prepareQC| E
     D[Did I get any prepareQCs?] --> |No| Z
 
-    E[Am I ahead of highPrepareQC?] --> |Yes| G
-    E[Am I ahead of highPrepareQC?] --> |No| Z
+    E[Am I ahead of prepareQC?] --> |Yes| G
```

Comment:
I thought we agree this should be `highPrepareQC` instead of `prepareQC`, no?

![Screenshot 2023-01-16 at 4 03 29 PM](https://user-images.githubusercontent.com/1892194/212765336-54802193-965a-43b3-a4ef-105c3928b835.png)